{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2023-50308",
    "ASSIGNER": "psirt@us.ibm.com"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "NVD-CWE-noinfo"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "https://www.ibm.com/support/pages/node/7105506",
        "name": "https://www.ibm.com/support/pages/node/7105506",
        "refsource": "",
        "tags": [
          "Patch",
          "Vendor Advisory"
        ]
      },
      {
        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/273393",
        "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/273393",
        "refsource": "",
        "tags": [
          "VDB Entry",
          "Vendor Advisory"
        ]
      },
      {
        "url": "https://security.netapp.com/advisory/ntap-20240307-0001/",
        "name": "https://security.netapp.com/advisory/ntap-20240307-0001/",
        "refsource": "",
        "tags": []
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "IBM Db2 for Linux, UNIX and Windows (includes DB2 Connect Server) 11.5 under certain circumstances could allow an authenticated user to the database to cause a denial of service when a statement is run on columnar tables.  IBM X-Force ID:  273393."
      }
    ]
  }
}